frontend/common/src/components/UserProfile/ProfileSections/RoleSalarySection.tsx

```diff
@@ -21,31 +21,30 @@ const RoleSalarySection: React.FunctionComponent<{
     return <span data-h2-font-color="b(red)">{msg}</span>;
   }
 
+  const anyCriteriaSelected =
```

Comment:
Maybe you want to use the lodash isEmpty here, too?  And in WorkLocation and WorkPreferences?